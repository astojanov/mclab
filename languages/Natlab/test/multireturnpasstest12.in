
a = foo();
[b(1),c(1)] = foo();
d = foo();