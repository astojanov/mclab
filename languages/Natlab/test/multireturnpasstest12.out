
a = foo();
[t1, t2] = foo();
b(1) = t1;
c(1) = t2;
d = foo();
